post_installation_configuration/machine-configuration-tasks.adoc

```diff
@@ -21,8 +21,7 @@ include::modules/checking-mco-status.adoc[leveloffset=+2]
 [id="using-machineconfigs-to-change-machines"]
 == Using `MachineConfig` objects to configure nodes
```

Comment:
s/`MachineConfig`/MachineConfig/ (We have decided not to use [backticks in headings](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings).)